backend/app/assets/javascripts/spree/backend/components/admin_nav.js

```diff
@@ -0,0 +1,25 @@
+Spree.ready(function() {
+  var adminNavToggle = document.querySelector("#admin-nav-toggle");
+  var adminNav =  document.querySelector(".admin-nav");
+
+  function showNav() {
+    adminNavToggle.innerHTML = '<i class=\"fa fa-chevron-left fa-2x\">';
```

Comment:
I don't think we need this change of innerHTML if we use CSS to transform the icon